<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayersManagerDialog</class>
 <widget class="QDialog" name="LayersManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../playground.qrc">
    <normaloff>:/images/document-multiple.png</normaloff>:/images/document-multiple.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="LayersListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="AddLayerPushButton">
           <property name="text">
            <string>&amp;Add layer</string>
           </property>
           <property name="icon">
            <iconset resource="../../playground.qrc">
             <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removePushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Remove layer</string>
           </property>
           <property name="icon">
            <iconset resource="../../playground.qrc">
             <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Layer propeties</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NameLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="TypeComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>NORMAL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TRANSPARENT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GUI</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../playground.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LayersManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LayersManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddLayerPushButton</sender>
   <signal>pressed()</signal>
   <receiver>LayersManagerDialog</receiver>
   <slot>addNewLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NameLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>LayersManagerDialog</receiver>
   <slot>setName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TypeComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>LayersManagerDialog</receiver>
   <slot>setType(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removePushButton</sender>
   <signal>pressed()</signal>
   <receiver>LayersManagerDialog</receiver>
   <slot>removeLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNewLayer()</slot>
  <slot>setName()</slot>
  <slot>setType(QString)</slot>
  <slot>removeLayer()</slot>
 </slots>
</ui>
